stripe/src/main/java/com/stripe/android/networking/AnalyticsRequestExecutor.kt

```diff
@@ -28,7 +27,6 @@ internal fun interface AnalyticsRequestExecutor {
          * @return the response status code. Used for testing purposes.
          */
         @VisibleForTesting
-        @Throws(APIConnectionException::class, InvalidRequestException::class)
```

Comment:
isn't it still throwing `APIConnectionException`?

Reply:
It is, but we always catch exceptions thrown from this method in `executeAsync`. We typically only annotate `@Throws` for checked exceptions.